### PR TITLE
Update iOS lower bound to iOS 12

### LINKED ISSUE #6: Drop iOS 10 support from browserslist-config
# Issue Description
With iOS 12 out, we should consider rolling iOS 10 off of our browserlists-config

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
With iOS 13 available, upping the lower bound for iOS to iOS 11

Closes #6

### CODE REVIEW

**Review by benbcai — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Comment by mjhenkes** [comment]
Opinions whether this should be a mvb?

**Comment by emilyrohrbough** [comment]
I feel like it kinda is... but not sure...the reason I haven't +1'd.

File: browserslist.config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module.exports = [
-  'iOS >= 10',
+  'iOS >= 12',
```
Comment (emilyrohrbough) [inline browserslist.config.js:2]: Why are we defining the iOS version as `iOS >= 12` instead of `last 2 iOS versions`?

**Comment by emilyrohrbough** [comment]
Currently this list includes:
Screen Shot 2019-11-06 at 12 11 32 PM


After this change:
Screen Shot 2019-11-06 at 12 11 24 PM

**Comment by emilyrohrbough** [comment]
Once this is merged, and we've upgraded to use it, we will need to update terra-ui: https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support

Additionally the ReadMe for this project might need updates: https://github.com/cerner/browserslist-config-terra#supported-browsers

File: browserslist.config.js (hunk 1 above)
Comment (emilyrohrbough) [reply browserslist.config.js:2]: nvm. this drops minor version released for iOS: https://browserl.ist/?q=last+2+iOS+versions%2C+last+2+and_chr+versions%2C+last+2+android+versions%2C+last+2+chrome+versions%2C+last+2+edge+versions%2C+last+2+firefox+versions%2C+last+2+ie+versions%2C+last+2+safari+versions

**Review by emilyrohrbough — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 -----------------
+## Breaking
+* Update iOS lower bound from iOS 10 to iOS 11
```
Comment (StephenEsser) [inline CHANGELOG.md]: This entry should be from 10 -> 12 based on the change below

**Review by StephenEsser — APPROVED** [review]

**Comment by bjankord** [comment]
LGTM

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,9 @@ ChangeLog
 
 Unreleased
 -----------------
+## Breaking
```
Comment (mjhenkes) [inline CHANGELOG.md]:
```suggestion
### Breaking
```

**Pull request merged**